receiver/elasticsearchreceiver/metadata.yaml

```diff
@@ -1 +1,272 @@
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+  elasticsearch.node.operations.completed:
+    description: Number of operations completed.
+    unit: "{operations}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [operation_type]
+  elasticsearch.node.operations.time:
+    description: Time spent on operations.
+    unit: ms
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [operation_type]
+  elasticsearch.node.shards.size:
+    description: Size of the shards assigned to this node.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+  elasticsearch.node.thread_pool.threads:
+    description: Number of the threads in the thread pool.
+    unit: "{threads}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [thread_state]
+  elasticsearch.node.thread_pool.queued_tasks:
+    description: Number of queued tasks in the thread pool.
+    unit: "{tasks}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [thread_pool_name]
+  elasticsearch.node.thread_pool.finished_tasks:
+    description: Number of tasks finished by the thread pool
+    unit: "{tasks}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [thread_pool_name, task_state]
+  elasticsearch.node.documents:
+    description: Number of documents on the node.
+    unit: "{documents}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [document_type]
+  elasticsearch.node.open_files:
+    description: Number of open file descriptors held by the node.
+    unit: "{files}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+  # these metrics are from /_cluster/health, and are cluster level metrics
+  elasticsearch.cluster.shards:
+    description: Number of shards in the cluster.
+    unit: "{shards}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [shard_type]
+  elasticsearch.cluster.data_nodes:
+    description: Number of data nodes in the cluster.
+    unit: "{nodes}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+  elasticsearch.cluster.nodes:
+    description: Total number of nodes in the cluster.
+    unit: "{nodes}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+  elasticsearch.cluster.health:
+    description: The health of the cluster.
+    extended_documentation: 
+      Health status is based on the state of its primary and replica shards. 
+      0 indicates green, 1 indicates yellow, and 2 indicates red.
```

Comment:
I'm not sure how we should model this, but there is at least one proposal [here](https://github.com/open-telemetry/oteps/pull/185/files) that suggests health metrics should be binary. Perhaps this should be omitted for this PR and added in another which may end up being a point of debate.

Reply:
I can omit for now; But maybe I misnamed it here, so let me get your opinion before I do.

It's from the [cluster health endpoint](https://www.elastic.co/guide/en/elasticsearch/reference/7.16/cluster-health.html#cluster-health-api-response-body), the `status` field. If this was `elasticsearch.cluster.status`, would it be more palatable, or do we still think we should discuss this more later?

Reply:
The problem is that it's not really making use of the metric data model in an intended way. It's an enumeration of values, with a mapping that must be interpreted through documentation. I suspect backends will be opinionated about this.

Perhaps @bogdandrutu or @jsuereth can provide guidance on this one.

Reply:
I'd also like to add that we do have similar things in other components, for instance, [k8sclusterreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8sclusterreceiver), where the condition metrics really are an enum (true, false, and unknown); So it's definitely something that has come up before and I'm sure it'll come up in the future.